app/controllers/api/v2/searches_controller.rb

```diff
@@ -93,21 +93,22 @@ def handle_query_routing
       def search_params
         @search_params ||= params.permit(:access_key,
```

Comment:
Alphabetized and left some in-line comments on existing params.  Only real change is addition of `:tags`.